mllib/src/main/scala/org/apache/spark/ml/classification/LogisticRegression.scala

```diff
@@ -218,31 +228,48 @@ class LogisticRegression(override val uid: String)
 
   override def getThreshold: Double = super.getThreshold
 
+  /**
+   * Whether to over-/undersamples each of training instance according to the given
```

Comment:
"Whether to over-/under-sample training instances according to the given weights in `weightCol`. If empty, all instances are treated equally (weight 1.0)."